python/semantic_kernel/agents/azure_ai/azure_ai_agent.py

```diff
@@ -122,7 +124,10 @@ def __init__(
 
     @classmethod
```

Comment:
This can actually be updated to a static method since we're not doing anything with `cls`.